t/27-consoles-sshIucvconn.t

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/perl
+
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::MockObject;
+use Test::MockModule;
+use Test::Warnings ':report_warnings';
+use Test::Output;
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::sshIucvconn;
+use testapi;
```

Comment:
If testapi is really needed in a test, then at least don't import anything, e.g.
```suggestion
use testapi ();
```
because you don't use any functions from it in the test.
testapi exports a lot of functions.

But I think it can be improved further.
`consoles::sshIucvconn` is doing `bmwqemu::diag()`, and that's the only reason why a `use testapi` fixes it. It would be enough and good practice to add a `use bmwqemu` in `consoles::sshIucvconn`.

And as a further improvement, `bmwqemu::diag` is now an alias for `log::diag`, so I suggest to do this in `consoles::sshIucvconn`:
```
use log qw(diag);
...
diag("ssh iucvconn: grabbing serial console for guest: $zvmguest");
```
Then the test wouldn't have to use testapi.

Reply:
I tried first suggestion. good point and worked ofc.
I do `use bmwqemu qw(diag fctwarn);`  now. 

I am not sure if you are asking to more tho.